### DISCUSSION TITLE
Is it usable ?

### DISCUSSION THREAD

**Discussion by cfab** [wp-graphql/wp-graphql-smart-cache#167] [General]
Hello Jason, 
Is there something particular I need to do in order to use the plugin ?
I downloaded the latest release and installed on a local site (local by flywheel), the zip contained in the archive. 
I can see the news tabs in wp-graphql UI settings (tabs for cache and saved queries) but when I test queries in the browser, like in your video, I don't see any change in terms of speed. 
In the headers, I can see that "access-control-max-age" reflects my settings, but regarding cache-control I still have "no-cache". Do I have to add some elements or code in order to make it work ? Is it related to the hosting server (wp-engine only) ?
Thank you !

**Comment by cfab** [comment]
Thank you for access to the docs. I understand better now ;-)

**Reply by jasonbahl** [reply]
@cfab We're still in beta, but the docs are now available in the repo here: https://github.com/wp-graphql/wp-graphql-smart-cache#readme

Would love to hear your thoughts after going through the docs.

**Comment by cfab** [comment]
Thanks !

**Reply by cfab** [reply]
As the host I usually work with, Infomaniak, is not (yet ?) compatible with the network cache part, I'm currently testing the object cache part and I can indeed witness 3 times faster request times. Hopefully I can test network cache soon.
Wondering if the benefit in terms of speed are similar or even better ?
Thank you.
